OxfordBodleian/Juel-Jensen/BDLaethe31.xml

```diff
@@ -50,35 +611,145 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <width>180</width>
+                                    </dimensions>
+                                    <dimensions type="margin" unit="mm" xml:id="margin1">
+                                        <dim type="right">10</dim>
+                                        <dim type="left">9</dim>
+                                        <dim type="top">17</dim>
+                                        <dim type="bottom">70</dim>
+                                        <dim type="intercolumn">13</dim>
+                                    </dimensions>
+                                    <ab type="pricking">Ruling is visible. Pricking is not visible.</ab>
+                                </layout>
+                                
+                                <layout columns="2" writtenLines="83 85">
+                                    <note>Number of characters per line: 13–17.</note>
+                                    <note>The outer margins are delimited by a decorated band.</note>
+                                    <dimensions unit="mm" xml:id="textarea2">
+                                        <height>625–643</height>
+                                        <width>172–180</width>
+                                    </dimensions>
+                                    <dimensions type="margin" unit="mm" xml:id="margin2">
+                                        <dim type="right">9</dim>
+                                        <dim type="left">8</dim>
+                                        <dim type="top">11</dim>
+                                        <dim type="bottom">12</dim>
+                                        <dim type="intercolumn">11</dim>
+                                    </dimensions>
+                                    <ab type="pricking">Ruling is visible. Pricking is not visible.</ab>
+                                </layout>
+                            </layoutDesc>
                         </objectDesc>
                         
                         <handDesc>
-                            <handNote xml:id="h1" script="Ethiopic">
-                           
-                            </handNote>
+                                <handNote xml:id="h1" script="Ethiopic">
+                                    <date notBefore="1800" notAfter="1899">Nineteenth century.</date>
+                                    <seg type="ink">Black, red.</seg>
+                                    <desc>Fine handwriting. Characters are uniformly spaced. The same hand wrote
+                                    <ref type="mss" corresp="UppEt45 UppEt46 UppEt47"/>.</desc>
+                                    <seg type="rubrication">Few lines of the incipits and other lines within texts; some occurrences of the names of Jesus Christ, 
+                                        Mary, Michael, Gabriel, Raphael, and Solomon; elements of punctuation and numerals; the names of the owners 
```

Comment:
we had previously agreed to summarize such cases as "some occurrences of holy names". ANd could you swap the position of "elements of punctuation and numerals" and "the names of the persons involved in the manuscript production"?